package/common_init/overlay/etc/init.d/S95watchdog_disable

```diff
@@ -10,4 +12,4 @@ stop() {
   :
 }
 
-source /etc/init.d/template_command.inc.sh
```

Comment:
why make this change?  I think source is easier to see.

Reply:
Basically to make the linter happy.  ShellCheck isn't able to detect that our shell variant (ash) understands `source` over `.` so it issues a warning.  We could just disable the warning too...

Reply:
I undid the `source` -> `.` change and just turned off that particular warning.